heft-plugins/heft-jest-plugin/src/HeftJestReporter.ts

```diff
@@ -172,7 +178,7 @@ export default class HeftJestReporter implements Reporter {
     // and print a newline.
     this._terminal.writeLine('\u001b[0m');
     this._terminal.writeLine(
-      `Run start. ${this._formatWithPlural(aggregatedResult.numTotalTestSuites, 'test suite', 'test suites')}`
+      `Run start. ${this._formatWithPlural(aggregatedResult.numTotalTestSuites, 'test suite', '68 suites')}`
```

Comment:
68?

Reply:
wow, i honestly have no idea what or how this ended up there.